managed-databases/postgresql-and-mysql/how-to/change-volume-type.mdx

```diff
@@ -37,7 +37,9 @@ It is possible to change your [volume type](/managed-databases/postgresql-and-my
     The Database Instance goes into the `Initializing` state.
 
     <Message type="important">
-      If you are changing from a Block to a local volume, make sure to select a node with  a local storage size greater than or equal to your current Block Storage.
+      - When switching from a local volume to a Block volume, and if [Autobackup](/managed-databases/postgresql-and-mysql/how-to/enable-autobackup) is enabled, be aware that the snapshot type will change. Local volumes utilize logical backups, whereas Block storage employs snapshots. Consequently, the billing for these two types will differ as well.
+      - Disk size cannot be reduced, so when switching from local to block, the volume size remains the same by default.
```

Comment:
Not sure if block should be capitalized here?

Reply:
```suggestion
      - Disk size cannot be reduced, so when switching from local to Block, the volume size remains the same by default.
```